reference/mongodb/mongodb/driver/monitoring/commandsubscriber.xml

```diff
@@ -45,6 +45,32 @@
 
   </section>
 
+  <section role="changelog">
+   &reftitle.changelog;
+   <para>
+    <informaltable>
+     <tgroup cols="2">
+      <thead>
+       <row>
+        <entry>&Version;</entry>
+        <entry>&Description;</entry>
+       </row>
+      </thead>
+      <tbody>
+       <row>
+        <entry>PECL mongodb 1.15.0</entry>
+        <entry>
+         Return types for methods are declared as tentative on PHP 8.0 and newer,
+         triggering deprecation notices in code that implements this interface
+         without declaring the appropriate return types. The <code>#[ReturnTypeWillChange]</code>
```

Comment:
`en/reference/mongodb/tutorial/apm.xml` has some code examples that implement CommandSubscriber. You may want to add `#[ReturnTypeWillChange]` attributes to relevant methods to be consistent with what the PHP docs do elsewhere (e.g. Iterator examples).